### PR TITLE
Addition: web resources hub project

### LINKED ISSUE #395: Addition: web resources hub
Add my project web resources hub [GSSoc'23]

### PR DESCRIPTION
**Pull request by jayk-gupta**
## Related Issue
closes #395 

## Description
Added my project web resources hub

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -58,6 +58,24 @@
         "tech": ["React", "CSS", "Node.js"]
       }
     ]
+  },
+   {
+    "github_username": "jayk-gupta",
+    "Social_media": {
+      "gitHub": "jayk-gupta",
```
Comment (priyankarpal) [inline src/DB/projects.json:65]: add the whole URL

File: src/DB/projects.json [hunk 2]
```diff
@@ -58,6 +58,24 @@
...
+      "LinkedIn": "",
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": "jayk_gupta"
```
Comment (priyankarpal) [inline src/DB/projects.json:69]: add the whole URL

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by jayk-gupta** [comment]
Done the changes!

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**